Completed view-products-for-sales-front-end | Peter Han

peterjhan88 opened the pull request:
Initial app.jsx changed to render Header and ProductList
custom styles.css added for fonts and sizes of div
unused class deleted
![image](https://user-images.githubusercontent.com/53169603/72292549-72af0d80-3606-11ea-9bcd-b90fef042919.png)

peterjhan88 committed:
Completed view-products-for-sales-front-end | Peter Han
Initial app.jsx changed to render Header and ProductList
custom styles.css added for fonts and sizes of div

peterjhan88 committed:
unused class deleted

thebearingedge commented on client/components/header.jsx:
```diff
@@ -0,0 +1,12 @@
+      <header className='bg-dark d-flex header-height flex-justify-start align-items-center'>
+        <div className='ml-5 mr-2 d-flex align-items-center text-weight-bold text-white header-logo'>$</div>
+        <div className='d-flex align-items-center text-white header-name'>Wicked Sales</div>
+      </header>
```
🤔 Can you simplify this? There are _a lot_ of CSS classes here.

peterjhan88 replied:
simplified by using custom css

thebearingedge commented on client/components/product-list-item.jsx:
```diff
@@ -0,0 +1,16 @@
+        <div className='card-body'>
+          <h5 className='card-title'>{this.props.productName || 'Product Name'}</h5>
+          <p className='card-text text-price text-weight-bold'>{`$ ${(this.props.price / 100).toFixed(2)}` || '0.00'}</p>
+          <p className='card-text'>{this.props.shortDescription || 'Short description sit amet, consectetur adipsicing elit...'}</p>
```
Are these `||` guards necessary?

peterjhan88 replied:
deleted

thebearingedge commented on client/components/product-list.jsx:
```diff
@@ -0,0 +1,44 @@
+    return (
+      <div className='col-12 d-flex flex-wrap justify-content-center bg-light'>
+        {allProducts}
+      </div>
```
Try using a `.row` here and then `.col-md-4` on each list item.

peterjhan88 replied:
changed as requested

thebearingedge commented on server/public/styles.css:
```diff
@@ -0,0 +1,20 @@
+* {
+  box-sizing: border-box;
+}
```
This `box-sizing` reset is already included with Bootstrap.

peterjhan88 replied:
deleted

peterjhan88 committed:
Changed as request
Used custom css to simplify the class for header-logo header-name
Deleted as requested
